src/main/java/com/labsynch/labseer/service/SaltFormServiceImpl.java

```diff
@@ -244,6 +250,8 @@ public SaltForm updateSaltForm(Parent parent, SaltForm saltForm, Set<IsoSalt> is
 					// numbering
 					lot.setCorpName(lotService.generateCorpName(lot));
 					newLotCorpName = lot.getCorpName();
+					assayService.renameBatchCode(oldLotCorpName, newLotCorpName, "SaltFormService");
```

Comment:
Do you know if this gets hit in the case that a lot hasn't changed names?  i.e. Do we need to test for equality and only call assay/container rename service if the lot corp name has changed?

Reply:
Ah thanks for pointing that out. I saw this was within the context of 

```
if (!propertiesUtilService.getCorpBatchFormat().equalsIgnoreCase("cas_style_format")) {
					// if NOT cas_style lot corpName; generate a new lot corp name based on lot
					// numbering
					lot.setCorpName(lotService.generateCorpName(lot));
```

but forgot there are common circumstances where the lot corp name doesn't change so I added a 

`if (!oldLotCorpName.equals(newLotCorpName)) {`

check.